resources/qml/Menus/PrinterMenu.qml

```diff
@@ -1,60 +1,89 @@
-// Copyright (c) 2018 Ultimaker B.V.
+// Copyright (c) 2022 Ultimaker B.V.
 // Cura is released under the terms of the LGPLv3 or higher.
 
 import QtQuick 2.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.4
 
-import UM 1.2 as UM
+import UM 1.6 as UM
 import Cura 1.0 as Cura
 
 Menu
 {
     id: menu
-//    TODO Enable custom style to the menu
-//    style: MenuStyle
-//    {
-//        frame: Rectangle
-//        {
-//            color: "white"
-//        }
-//    }
-
-    MenuItem
+
+    UM.MenuItem
     {
+        id: networkEnabledPrinterItem
         text: catalog.i18nc("@label:category menu label", "Network enabled printers")
         enabled: false
-        visible: networkPrinterMenu.count > 0
+        visible: networKPrinterInstantiator.count > 0
     }
 
-    NetworkPrinterMenu
+    Instantiator
     {
-        id: networkPrinterMenu
+        id: networKPrinterInstantiator
+        model: Cura.GlobalStacksModel {filterOnlineOnly: true}
+        UM.MenuItem
+        {
+            property string connectGroupName:
+            {
+                if("group_name" in model.metadata)
+                {
+                    return model.metadata["group_name"]
+                }
+                return ""
+            }
+            text: connectGroupName
+            checkable: true
+            checked: Cura.MachineManager.activeMachineNetworkGroupName == connectGroupName
+            onTriggered:
+             {
+             print(typeof(model.id))
+                Cura.MachineManager.someFunction("YAY")
```

Comment:
Missed this last time but is this still some debugging code?

Reply:
Yep! Fixed in 94c006009b8aa05a9ad52e8352752ab22f59af7f